Add information about Secure Storage in Studio 7.11 (#267)
mulesoft/docs-studio: modules/ROOT/pages/whats-new-in-studio.adoc

= What’s New in Studio

> Anypoint Studio 7.11 introduces support for Mule runtime engine version 4.4. It also enables you to encrypt and save your credentials using a Secure Storage facility.

Reviewer: ```suggestion
Anypoint Studio 7.11 introduces support for Mule runtime engine version 4.4. Studio 7.11 also enables you to encrypt and save your credentials by introducing Secure Storage.
```

> == Secure Storage

Reviewer: Consider rewording as suggested in the edits to ...secure-storage.adoc.
